api/catalog/api/models/audio.py

```diff
@@ -96,6 +97,40 @@ class Meta:
         abstract = True
 
 
+class AudioAddOn(OpenLedgerModel):
+    audio_identifier = models.UUIDField(
```

Comment:
If this is the primary key of this model we should drop the `audio_` prefix.

Reply:
Should also note that leaving it as it is wouldn't be wrong either, as it refers to the Audio model instance.